docs/performance_tuning.md

```diff
@@ -0,0 +1,285 @@
+    - If your model does not come from a supported backend, you can look into the [Python Backend](https://github.com/triton-inference-server/python_backend) or writing a [Custom C++ Backend](https://github.com/triton-inference-server/backend/blob/main/examples/README.md) to support your model. 
+    The Python Backend provides a simple interface to execute requests through a generic python script, but may not be as performant as a Custom C++ Backend. 
+    Depending on your use case, the Python Backend performance may be a sufficient tradeoff for the simplicity of implementation.
+
+2. Can I run inference on my served model?
+    - Assuming you were able to load your model on Triton, the next step is to verify that we can run inference requests and get a baseline performance benchmark of your model. 
+    Triton's [Perf Analyzer](https://github.com/triton-inference-server/server/blob/main/docs/perf_analyzer.md) tool specifically fits this purpose. 
+    Here is a simplified output for demonstration purposes:
+
+    ```
+    # NOTE: "my_model" represents the model currently being served by 
+    #       Triton in the background from the previous step.
+
+    $ perf_analyzer -m my_model
+
+    ...
+
+    Inferences/Second vs. Client Average Batch Latency
+    Concurrency: 1, throughput: 482.8 infer/sec, latency 12613 usec
+    ```
+
+    - This gives us a sanity test that we are able to successfully form input requests and receive output responses to communicate with the model backend via Triton APIs. 
+    If Perf Analyzer fails to send requests and it is unclear from the error how to proceed, then you may want to sanity check that your model `config.pbtxt` inputs/outputs match what the model expects. 
+    If the config is correct, check that the model runs successfully using its original framework directly. 
+    If you don't have your own script or tool to do so, [Polygraphy](https://github.com/NVIDIA/TensorRT/tree/main/tools/Polygraphy) is a useful tool to run sample inferences on your model via various frameworks. 
+    Currently, Polygraphy supports ONNXRuntime, TensorRT, and TensorFlow 1.x.
+    - The definition of "performing well" is subject to change for each use case. 
+    Some common metrics are throughput, latency, and GPU utilization. 
+    There are many variables that can be tweaked just within your model configuration (`config.pbtxt`) to obtain different results.
+    - As your model, config, or use case evolves, [Perf Analyzer](https://github.com/triton-inference-server/server/blob/main/docs/perf_analyzer.md) is a great tool to quickly verify model functionality and performance.
+
+3. How can I improve my model performance?
+    - To further understand the best model configuration you can provide to Triton for your use case, Triton's [Model Analyzer](https://github.com/triton-inference-server/model_analyzer) tool can help. 
```

Comment:
One more sentence after this to give a high level overview of what MA does would be helpful.  Something like "Model Analyzer can automatically or manually search through config combinations to find the optimal triton configuration to meet your constraints.